### DISCUSSION TITLE
"cancel" with color?

### DISCUSSION THREAD

**Discussion by stapeleliz** [KaTeX/KaTeX#3535] [Q&A]
I like to use color, when I can, to highlight what factors, exactly, are being cancelled by which other factors. When there is a lot of cancellation going on, color can help clarify what is going on. One example (taken from my website) would be this:

![mult17](https://user-images.githubusercontent.com/31939907/151866579-d5c2ee70-ef91-418a-b353-4853d6787f6f.gif)

The above is an image, which of course screen-readers can't see. I do the best I can with the "alt" tags.

I can see that this colored-cancellation functionality is apparently available within TeX from a discussion here:

https://tex.stackexchange.com/questions/445945/how-to-strike-through-or-cancel-using-different-colors-in-the-same-line

I see from a discussion here that there will probably not be a "cancelto" function added to KaTeX:

https://github.com/KaTeX/KaTeX/discussions/2560

...but I'm willing to try the "kludge" provided at the end of that thread. However, I cannot see any current means of doing color-based cancellation. Am I missing something, or maybe using a current functionality incorrectly? If not, would this functionality possibly be added in the near-ish future?

Thank you!

Eliz.

**Comment by edemaine** [comment]
You can color `\cancel` using `\color` and resetting color inside the `\cancel`:

![image](https://user-images.githubusercontent.com/2218736/151868284-ac9d0e03-55d4-4080-93ea-07a75ffa6669.png)

```latex
\color{purple}\cancel{\color{black}x+y}
```

**Comment by stapeleliz** [comment]
I'll give that a try. Thank you!
